<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newimages/newimages/logo1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="ClickableLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;首页&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="bottomImgLabel_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>921</width>
     <height>471</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newimages/newimages/background.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="loginDialogWidget" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 8px; 
box-shadow: 0 0 20px rgba(0,0,0,0.1); </string>
   </property>
   <widget class="ClickableLabel" name="switchLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #0066cc; text-decoration: underline; cursor: pointer;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aaff;&quot;&gt;验证码登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>124</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>阅读并同意 </string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; color:#00aaff;&quot;&gt;服务协议&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; color:#000000;&quot;&gt;和&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; color:#00aaff;&quot;&gt;个人信息保护政策&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="tip4Label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
    </property>
    <property name="text">
     <string>请阅读并同意用户协议和隐私政策！</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="registerLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aaff;&quot;&gt;免费注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="loginStackWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="tip2Label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
      </property>
      <property name="text">
       <string>请输入密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="tip1Label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
      </property>
      <property name="text">
       <string>请输入用户名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pwdEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #cccccc; /* 浅灰色边框（默认可见） */
    border-radius: 4px; /* 可选：圆角更美观 */
    padding: 0 10px; /* 可选：输入内容不贴边 */
    height: 40px; /* 可选：统一高度 */
}
QLineEdit:focus {
    border-color: #0066cc; /* 聚焦时变蓝色边框（更醒目） */
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>登录密码</string>
      </property>
     </widget>
     <widget class="ClickableLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;忘记密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 登录按钮基础样式（橙色边框+橙色背景，搭配白色文字） */
QPushButton {
    /* 橙色边框：#ff7d00 是常用登录页橙色，可替换为 #ff9900（浅橙）、#ff6600（深橙） */
    border: 2px solid #ff7d00; 
    border-radius: 4px; /* 圆角（和输入框保持一致，更协调） */
    background-color: #ff7d00; /* 按钮背景色（和边框同色，一体化效果） */
    color: white; /* 文字白色 */
    font-size: 16px; /* 文字大小 */
    height: 44px; /* 按钮高度（和输入框呼应） */
}

/* 悬浮状态（边框加深+背景色变深，更有交互感） */
QPushButton:hover {
    border-color: #ff6600; /* 悬浮时边框深橙色 */
    background-color: #ff6600; /* 悬浮时背景深橙色 */
}

/* 点击状态（边框再加深+背景色压暗，模拟按压效果） */
QPushButton:pressed {
    border-color: #e65c00; /* 点击时边框更深橙色 */
    background-color: #e65c00; /* 点击时背景更深橙色 */
}

/* 禁用状态（可选：如果按钮需要禁用，边框和背景变灰色） */
QPushButton:disabled {
    border-color: #cccccc;
    background-color: #cccccc;
    color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #cccccc; /* 浅灰色边框（默认可见） */
    border-radius: 4px; /* 可选：圆角更美观 */
    padding: 0 10px; /* 可选：输入内容不贴边 */
    height: 40px; /* 可选：统一高度 */
}
QLineEdit:focus {
    border-color: #0066cc; /* 聚焦时变蓝色边框（更醒目） */
}</string>
      </property>
      <property name="placeholderText">
       <string>国内手机号/用户名/邮箱/卡号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#000000;&quot;&gt;账号密码登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#000000;&quot;&gt;验证码登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phoneEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #cccccc; /* 浅灰色边框（默认可见） */
    border-radius: 4px; /* 可选：圆角更美观 */
    padding: 0 10px; /* 可选：输入内容不贴边 */
    height: 40px; /* 可选：统一高度 */
}
QLineEdit:focus {
    border-color: #0066cc; /* 聚焦时变蓝色边框（更醒目） */
}</string>
      </property>
      <property name="placeholderText">
       <string>请输入手机号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="codeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #cccccc; /* 浅灰色边框（默认可见） */
    border-radius: 4px; /* 可选：圆角更美观 */
    padding: 0 10px; /* 可选：输入内容不贴边 */
    height: 40px; /* 可选：统一高度 */
}
QLineEdit:focus {
    border-color: #0066cc; /* 聚焦时变蓝色边框（更醒目） */
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>短信验证码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 登录按钮基础样式（橙色边框+橙色背景，搭配白色文字） */
QPushButton {
    /* 橙色边框：#ff7d00 是常用登录页橙色，可替换为 #ff9900（浅橙）、#ff6600（深橙） */
    border: 2px solid #ff7d00; 
    border-radius: 4px; /* 圆角（和输入框保持一致，更协调） */
    background-color: #ff7d00; /* 按钮背景色（和边框同色，一体化效果） */
    color: white; /* 文字白色 */
    font-size: 16px; /* 文字大小 */
    height: 44px; /* 按钮高度（和输入框呼应） */
}

/* 悬浮状态（边框加深+背景色变深，更有交互感） */
QPushButton:hover {
    border-color: #ff6600; /* 悬浮时边框深橙色 */
    background-color: #ff6600; /* 悬浮时背景深橙色 */
}

/* 点击状态（边框再加深+背景色压暗，模拟按压效果） */
QPushButton:pressed {
    border-color: #e65c00; /* 点击时边框更深橙色 */
    background-color: #e65c00; /* 点击时背景更深橙色 */
}

/* 禁用状态（可选：如果按钮需要禁用，边框和背景变灰色） */
QPushButton:disabled {
    border-color: #cccccc;
    background-color: #cccccc;
    color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="ClickableLabel" name="getCode">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>170</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aaff;&quot;&gt;发送验证码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="tip3Label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
      </property>
      <property name="text">
       <string>请输入手机号</string>
      </property>
     </widget>
     <widget class="QLabel" name="tip5Label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
      </property>
      <property name="text">
       <string>手机号格式错误（11位有效号码）</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>管理员登录</string>
    </property>
   </widget>
   <zorder>loginStackWidget</zorder>
   <zorder>switchLabel</zorder>
   <zorder>radioButton</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>tip4Label</zorder>
   <zorder>registerLabel</zorder>
   <zorder>checkBox</zorder>
  </widget>
  <zorder>bottomImgLabel_2</zorder>
  <zorder>loginDialogWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
    <slot>on_switchLabel_clicked()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
